packages/deploy-tooling/test/unit/base/validate.test.ts

```diff
@@ -447,4 +453,31 @@ describe('deploy-test validation', () => {
             );
         });
     });
+    describe('Validate show additional info', () => {
+        const destinationsMock = { 'ABC123': {} };
+        test('show additional info ', async () => {
```

Comment:
Might be worth making these tests into a list since they all do the same thing except the result is different.